src/node/build/bundle.ts

```diff
@@ -68,6 +69,7 @@ export async function bundle(
         // other
         preserveEntrySignatures: 'allow-extension',
         output: {
+          sanitizeFileName,
```

Comment:
This isn't necessary as the function essentially is same as what rollup has by default. Still adding this for future (or in case there is some delay with us modifying this after rollup has -- shouldn't happen though, update frequency of this file is very less).